compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -303,7 +303,7 @@ error.additional.positional.arg.in.error.constructor=additional positional arg i
 error.missing.arg.within.parenthesis=missing arg within parenthesis
 error.missing.variable.name=missing variable name
 error.missing.positional.arg.in.error.constructor=missing positional arg in error constructor
-error.open.record.cannot.contain.rest.field=open record cannot contain rest field
+error.inclusive.record.type.descriptor.cannot.contain.rest.field=inclusive record type descriptor cannot contain rest field
```

Comment:
inclusive record type descriptor cannot contain "a" rest field?
I guess it is okay to not have "a", "an", "the" in enum name to keep it in the simplest form.